src/client/tile.cpp

```diff
@@ -36,10 +36,10 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 
 
 #if ENABLE_GLES
-#ifdef _IRR_COMPILE_WITH_OGLES1_
-#include <GLES/gl.h>
-#else
+#ifdef _IRR_COMPILE_WITH_OGLES2_
 #include <GLES2/gl2.h>
+#elif _IRR_COMPILE_WITH_OGLES1_
+#include <GLES/gl1.h>
```

Comment:
the order of these is irrelevant, this should be left as before
Also the path is `GLES/gl.h` not `GLES/gl1.h`

Reply:
I was getting a build error (header could not be found) without this

Reply:
Does Android not have `GLES/gl.h`? When I checked the NDK I have installed I could find one.

Reply:
How can we use GLES2 with GLES1 header?

Reply:
The code in this file just calls `glGetString`, which exists in GLESv1. Including the GLESv2 header does not make a difference.